plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -208,6 +208,7 @@ public static void main(String[] args)
             // Please set Delta Lake connector properties via VM options. e.g. -Dhive.metastore=glue -D..
             QueryRunner queryRunner = builder()
                     .addCoordinatorProperty("http-server.http.port", "8080")
+                    .addDeltaProperty("hive.metastore", System.getProperty("hive.metastore"))
```

Comment:
Per the code comment above, this supposedly already work for this property and others.

BTW I think we should consider deleting the DeltaLakeExternalQueryRunnerMain class. One can use development server with custom delta.properties for the same purpose.

Reply:
I prefer this query runner to development server because it doesn't require rebuild after changed the code.